src/nxt_application.h

```diff
@@ -49,7 +49,6 @@ typedef struct {
 typedef struct {
     char                       *home;
     nxt_conf_value_t           *path;
-    char                       *callable;
```

Comment:
This should go to a separate commit that removes dead code.

Reply:
I created a new commit for removing code